### ISSUE TITLE
Portable Enhancements broke window position

### ISSUE THREAD

**Issue by bcssov** [dnGrep/dnGrep#471]
The recently implemented portable enhancements broke the window position and dnGrep can't seem to restore the original settings. 

Steps to reproduce:
1. Use Universal extractor to extract files from the installer
2. Run the program. Everything works as expected, program launches it creates the local settings file
![1](https://user-images.githubusercontent.com/5864390/103464705-04db8480-4d36-11eb-9de7-c97b9edc5b19.png)
3. Close the program
4. Run the program again and the settings are not restored and program starts in top left bottom corner of my screen
![2](https://user-images.githubusercontent.com/5864390/103464708-0c9b2900-4d36-11eb-92ae-70f3b01e8a7c.png)

Additional info:
I just started the program and closed it in between sessions.

**Comment by doug24** [comment]
What version of Windows are you on?  Is this on Windows 7?  I've tested both a regular installation and a portable version on Windows 10 (build 2004) and it is working for me.

**Comment by bcssov** [comment]
Win 10 20H2. Please note that the build v2.9.282.0 (beta) works.

**Comment by bcssov** [comment]
Had to go into your code to see what is the problem, anyways the problem is not OS related but locale related. When you save settings into the settings file you did not take into account that delimiters can vary from culture to culture.

```  <item key="MainWindowBounds">20;20;1200;800</item>```

**Comment by doug24** [comment]
Ah, awesome, thanks.  I did see the delimiters in the another post, but I assumed that write and parse would use the same delimiters.

**Comment by bcssov** [comment]
Just to assure you here's what's in my %localappdata% directory:
``` 
            <setting name="MainFormExBounds" serializeAs="String">
                <value>585,298,1200,800</value>
            </setting>
```

**Comment by doug24** [comment]
I have a pull request pending, but would like someone to test this first.  Can you get to this url for build [v2.9.301](https://ci.appveyor.com/project/JVimes/dngrep/builds/37080365/job/wpjwq9ufqp2s34sx/artifacts)?   The msi is at the bottom of that page.

**Comment by bcssov** [comment]
It works, you can close this ticket.

**Comment by doug24** [comment]
Released in [v2.9.304](https://github.com/dnGrep/dnGrep/releases/tag/v2.9.304.0)
